### DISCUSSION TITLE
Manager assignments for the first authors

### DISCUSSION THREAD

**Discussion by ajshajib** [AstroBridge/BDLensing#209] [General]
As announced in #208, the group of first authors will manage 3-4 systems each. Let's discuss and arrange the managers for the 20 systems in this thread.

The responsibilities of the manager are:
- Keep a check if the modeling is progressing in a timely manner to meet the deadlines, and provide reminders to the modeler.
- Keep a check if the associated PRs are progressing on time, and provide necessary reminders to the modeler and reviewers. However, it is primarily the modeler's responsibility to remind the reviewers to provide reviews and update the PR promptly as necessary. The manager should check if the modeler hasn't been proactive in getting the PR merged on time.
- If a modeler has stopped working on a system, the manager would need to model the system themselves or find another willing modeler.

(I will be updating this table as the project progresses further.)

System | Modeler | Manager | Modeling done
--- | --- | --- | ---
DESIJ0132-1600 |  @Rajit13 (prev: @anikbiswas730)  | @Rajit13 | :white_check_mark:
DESIJ0136-0008 |  @SulymanHossain (prev: @AkbarAC)  | @Rajit13 |  :white_check_mark:
DESIJ0201-2739 |   @rafee-adnan  | @rafee-adnan | :white_check_mark:
DESIJ0215-2909 |   @numjahan | @rafee-adnan | :white_check_mark:
DESIJ0618+5018 |  @aaimtiaz (prev: @ajmi-nishuti) | @aaimtiaz | :white_check_mark:
DESIJ0923+1829 |   @seekermind (prev: @sourabh221b) | @mamun-nahid | :white_check_mark:
DESIJ1018-0008 |   @seekermind | @ZareefJafar | :negative_squared_cross_mark:
DESIJ1018-0121 |  @JobairHasan  | @JobairHasan | :white_check_mark:
DESIJ1101-0602 |  @rafee-adnan (prev: @Nishijannat) | @rafee-adnan | :white_check_mark:
DESIJ1205+4110 |  @SulymanHossain (prev: @mirsazzathossain) | @SulymanHossain | :white_check_mark:
DESIJ1225-0725 |  @SulymanHossain  | @SulymanHossain | :negative_squared_cross_mark:
DESIJ1421-0022 |   @SamWGamgee | @ZareefJafar | :negative_squared_cross_mark:
DESIJ1501+5208 |   @mamun-nahid  | @mamun-nahid | :white_check_mark:
DESIJ1537+1443 |   @Conjecture13  | @SulymanHossain |  :white_check_mark:
DESIJ1539+1650 |  @SulymanHossain (prev: @CHOC4HOL1C) | @SulymanHossain | :negative_squared_cross_mark:
DESIJ1554+0443 |  @ZareefJafar | @ZareefJafar | :white_check_mark:
DESIJ1624+0129 |  @aaimtiaz | @aaimtiaz | :white_check_mark:
DESIJ1709+3154 |  @TwistNemo | @mamun-nahid | :white_check_mark:
DESIJ2158+0257 |  @Rajit13  | @Rajit13 | :white_check_mark:
DESIJ2318-1106 |  @JobairHasan (prev: @rahulsust) | @JobairHasan | :negative_squared_cross_mark:

Writing manager: @rafee-adnan, @aaimtiaz

**Comment by ajshajib** [comment]
@rafee-adnan, @ZareefJafar, @Rajit13, @JobairHasan, @SulymanHossain, @aaimtiaz, please let me know which systems you would like to be assigned as the manager.

**Reply by aaimtiaz** [reply]
I'll let you know by the 6th Sep. If it's okay.

**Reply by ajshajib** [reply]
I have already assigned yours, @aaimtiaz. Hopefully, this will work for you.

**Reply by aaimtiaz** [reply]
I will try my best.

**Comment by Rajit13** [comment]
I'll help @AkbarAC @anikbiswas730 @Nishijannat as I've previously worked with these models at some point. An additional choice would be @TwistNemo's model.

**Reply by TwistNemo** [reply]
I'm currently working on my model, and my MCMC simulation is in progress.

**Reply by ajshajib** [reply]
I have assigned DESIJ1101-0602 to @rafee-adnan and DESIJ1709+3154 to @Rajit13 as managers.

**Reply by ajshajib** [reply]
I have reassigned DESIJ1709+3154 management to @mamun-nahid for equal distribution of the management tasks.

**Comment by SulymanHossain** [comment]
My preference: @mamun-nahid, @Conjecture13, @TwistNemo(if it's available)

**Reply by ajshajib** [reply]
I have assigned DESIJ1709+3154 to @Rajit13 for now, but if some rearrangement in the management tasks is necessary, I will keep your preference for this system in mind.

**Reply by SulymanHossain** [reply]
Bhaia, @CHOC4HOL1C just expressed his interest in working with me. So, can you please replace one previously assigned system with this?

**Comment by ZareefJafar** [comment]
I did not work with anyone else's system before, no good idea which one I should choose. Selecting **DESIJ0215-2909**, **DESIJ0923+1829** systems as a starting point as no one is assigned to them right now. Maybe **DESIJ1421-0022** also.

**Reply by ajshajib** [reply]
@rafee-adnan is assigned as the manager for DESIJ0215-2909, but I am assigning you the other two.

**Reply by ajshajib** [reply]
I have rearranged the table a bit, please check it again now for your management assignments.

**Reply by ZareefJafar** [reply]
Got it, thanks for the update

**Reply by seekermind** [reply]
I'm doing DESIJ0923+1829.
